packages/components/organisms/f-checkout/src/services/tests/analytics.test.js

```diff
@@ -0,0 +1,179 @@
+import Trak from '@justeat/f-trak';
+import { trackInitialLoad, trackFormInteraction, mapFieldNames } from '../analytics';
+import {
+    analyticsData
+} from '../../components/_tests/helpers/setup';
+
+describe('Checkout Analytics', () => {
+    let trackEventSpy;
+
+    beforeEach(() => {
+        trackEventSpy = jest.spyOn(Trak, 'event').mockImplementation();
+    });
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    describe('trackInitialLoad :: ', () => {
+        it('should call `event` method of `f-trak`', () => {
+            // Arrange
+            const expectedEvent = {
+                custom: {
```

Comment:
Could save declaring this object twice by putting it up on line 20 as a let and then just amending the `pageData` part for the tests on line 47 (as it looks like the same object for both sets of tests otherwise)

Reply:
Good point, thanks